src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -199,12 +199,15 @@
                                   :on-success (:on-submission-success props)})])
         (create-section-header "Method Configuration Name")
         (create-section
-         (if editing?
-           (style/create-text-field {:ref "confname" :style {:maxWidth 500}
-                                     :data-test-id "edit-method-config-name-input"
-                                     :defaultValue (:name config)})
-           [:div {:style {:padding "0.5em 0 1em 0"}
-                  :data-test-id "method-config-name"} (:name config)]))
+          (if editing?
+            [:div {}
+             (style/create-text-field {:ref "confname" :style {:width "100%"}
```

Comment:
@zarsky-broad Would you prefer something else in lieu of `{:width "100%"}`?

Reply:
try this:
```cljs
{:maxWidth 500
 :width "100%"
 :fontSize "100%"}
```